### ISSUE TITLE
Fix metadata

### ISSUE THREAD

**Issue by ProbablePrime** [Yellow-Dog-Man/Elements.Quantity#9]
Based on comment here: https://github.com/Yellow-Dog-Man/Elements.Quantity/pull/5

**Comment by Redd56** [comment]
Would this be counted as done since its in the build?

**Comment by ProbablePrime** [comment]
No: https://github.com/Yellow-Dog-Man/Elements.Quantity/commit/70ef5c39ba5dc41819e8989f8bc79707ab1f7a95 has not been re-done.
